src/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkActionsSaveButton.tsx

```diff
@@ -98,6 +114,11 @@ export const ArtworkActionsSaveButtonFragmentContainer = createFragmentContainer
           isAuction
           isClosed
         }
+        collectorSignals {
+          auction {
+            lotWatcherCount
+          }
+        }
         ...ArtworkActionsWatchLotButton_artwork
```

Comment:
This has me wondering if the number should just be part of the `ArtworkActionsWatchLotButton` component so the count can be  included in the update automatically - might make relay updates easier too.